Incidents.cs

```diff
@@ -37,8 +37,8 @@ public partial class Incidents
         [JsonConverter(typeof(ParseStringConverter))]
         public long? Ucr { get; set; }
 
-        [JsonProperty("dst")]
-        public DstUnion Dst { get; set; }
+        [JsonProperty("district")]
+        public DistrictUnion District { get; set; }
```

Comment:
Adding  the justify-content-center class to center the images on the page.

Reply:
Yes, very good idea.  Variable names should be self-describing.
Avoid the temptation to use abbreviations.  You'll forget what abbreviation you used!